core/menu.ts

```diff
@@ -84,7 +84,8 @@ export class Menu {
    */
   render(container: Element): HTMLDivElement {
     const element = document.createElement('div');
-    element.className = 'blocklyMenu blocklyNonSelectable';
+    // goog-menu is deprecated, use blocklyMenu.  May 2020.
+    element.className = 'blocklyMenu goog-menu';
```

Comment:
Looks like you might have picked up an old change in your rebase, could you remove the `goog-menu` class and the comment from here?
```suggestion
    element.className = 'blocklyMenu';
```